### PR TITLE
DTSSTCI-73

### PR DESCRIPTION
**Pull request by tomxelliott**
### Change description
Set up language toggle for notifications English/Welsh

### JIRA link
https://tools.hmcts.net/jira/browse/DTSSTCI-73

### CODE REVIEW

File: charts/sptribs-frontend/values.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ nodejs:
     IDAM_API_URL: 'https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net/o/token'
     REDIS_HOST: 'sptribs-{{ .Values.global.environment }}.redis.cache.windows.net'
     EQUALITY_URL: 'https://pcq.{{ .Values.global.environment }}.platform.hmcts.net'
-    CASE_API_URL: 'http://sptribs-case-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal'
+    CASE_API_URL: 'https://sptribs-case-api-pr-1810.preview.platform.hmcts.net'
```
Comment (rajaram-nagarajan) [inline charts/sptribs-frontend/values.yaml:19]: We just need to remember to revert this change while releasing the code changes.

**Review by rajaram-nagarajan — APPROVED** [review]
LGTM

File: charts/sptribs-frontend/values.yaml (hunk 1 above)
Comment (tomxelliott) [reply charts/sptribs-frontend/values.yaml:19]: Yeah will do, leaving it there for testing

**Comment by tomxelliott** [comment]
Note to self: remember to revert chart and values.yaml changes made for testing purposes

File: charts/sptribs-frontend/values.yaml (hunk 1 above)
Comment (pr00279) [reply charts/sptribs-frontend/values.yaml:19]: Does this still need to be kept for testing?

**Review by gmmagruder — APPROVED** [review]

**Review by gmmagruder — APPROVED** [review]

**Review by oleon23 — APPROVED** [review]
